modules/ROOT/pages/push/binding.adoc

```diff
@@ -28,34 +28,37 @@ include::{partialsdir}/generic-binding.adoc[]
 When you bind a mobile client to the {unifiedpush-service} service, a variant is created to store the credentials required to communicate with a push network.
 Those credentials need to be provided in the binding dialog.
 
-First pick your *Mobile Client Type*. Depending on the selection you then have to fill out the fields for either Android or iOS.
+Pick a *Mobile Client Type*. Fill out the required fields:
 
 [role="primary"]
 .Android
 
 ****
-You need to provide a *Server Key* and a *Sender ID*. Both can be found in the Firebase console under *Project Settings > Cloud Messaging*. Make sure that you have selected `Android` in the `Mobile Client Type` dropdown.
+- Provide a *Server Key* and a *Sender ID*. Both can be found in the Firebase console
+- Ensure that you have selected `Android` in the `Mobile Client Type` dropdown menu.
 ****
 
 [role="secondary"]
 .iOS
 
 ****
-You need an APNs client TLS certificate. Have a look at link:apple-setup.html[APNs Setup] for instructions on how to obtain one. Make sure that you have selected `iOS` in the `Mobile Client Type` dropdown.
-The certificate needs to be encoded in base64 before copying it into the field labelled `iOS .p12 file`. If you're on Mac OS run:
-
+- Ensure that you have selected `iOS` in the `Mobile Client Type` dropdown menu.
+- Acquire an APNs client TLS certificate. Refer to link:https://help.apple.com/developer-account/#/dev82a71386a[official Apple guide]. The certificate needs to be encoded in base64 before copying it into the field labeled `iOS .p12 file`. 
++
+If you are using macOS, run:
++
 ----
 cat <path/to/your/.p12> | base64
 ----
-
-If you're on Linux chances are that you're using a version of the base64 tool that does line wrapping. You need to disable this:
-
++
+If you are using Linux, chances are that you are using a version of the base64 tool that does line wrapping. You need to disable this:
```

Comment:
think it's better to give command and note after command:

NOTE: `--wrap=0` disables line wrapping